renovate.json

```diff
@@ -1,5 +1,32 @@
 {
   "$schema": "https://docs.renovatebot.com/renovate-schema.json",
-  "extends": ["config:base"],
-  "enabledManagers": ["docker-compose","maven-wrapper","gradle-wrapper"]
+  "extends": [
+    "config:base",
+    ":separateMultipleMajorReleases",
```

Comment:
This one is needed for the Docker image digests config?

Reply:
Not related to docker digest pinning, but this is a more sane setting when the current version is multiple majors behind. It will create PR's for all major versions in between, instead of only the latest. 
Mostly useful for software that has certain versions that are more special (LTS) like Java or Ubuntu.

Reply:
For in the readme?